WordPress/Classes/Utility/Media/MediaAssetExporter.swift

```diff
@@ -80,7 +81,14 @@ class MediaAssetExporter: MediaExporter {
                 return exportGIF(forAsset: asset, resource: resource, onCompletion: onCompletion, onError: onError)
             }
         }
-
+        // Sergio Estevao: If the image is of the type HEIC we make sure the exportImageType uses JPG,
+        // because WordPress doesn't accept HEIC.
+        if #available(iOS 11.0, *) {
+            if let resource = resources.first,
+                UTTypeEqual(resource.uniformTypeIdentifier as CFString, AVFileType.heic.rawValue as CFString) {
```

Comment:
I see other values of `AVFileType` that could be problematic (e.g. `dng`, `tiff`, `heif`,...).
Is this required for this fix? If not, can we split into a different PR and explore an option that always ensures a supported format (jpg/png/gif)?

Reply:
It's needed because by default photos are now saved in this format on iOS 11. When you try to use the code the way it was, it tried to export them to WP servers on that format, and failed miserably. 
The other formats could also be problematic, but they are not "common" use on iOS like HEIC is at the moment.